py/src/braintrust/dev/server.py

```diff
@@ -0,0 +1,165 @@
+from dataclasses import asdict
+from typing import Annotated, Any, List, Optional, Union, cast
+
+import uvicorn
+from fastapi import FastAPI
```

Comment:
this might be a good use case for bottle. we could do this with no extra dependencies (other than perhaps cors?) and even check a copy into the repo. not mandatory, just food for thought. 

https://github.com/bottlepy/bottle

Reply:
fastapi requires pydantic. pydantic is an untenable dependency because of the v1 / v2 incompatability. therefore, it's probably not a good idea to depend on fastapi (maybe if used as an optional dependency, but probably not)